core/runner.js

```diff
@@ -154,37 +154,21 @@ class Runner {
 
     /** @type {Array<LH.Result.LhrEntity>} */
     const entities = [];
-    /** @type {Record<string, number>} */
-    const entityIndexByOrigin = {};
-    /** @type {Record<string, number>} */
-    const entityIndexByName = {};
-
     for (const [entity, entityUrls] of classifiedEntities.urlsByEntity) {
       /** @type {LH.Result.LhrEntity} */
       const shortEntity = {
         name: entity.name,
         homepage: entity.homepage,
+        origins: [...new Set(
+          [...entityUrls].map(url => UrlUtils.getOrigin(url) || '').filter(Boolean))],
```

Comment:
nit: this is kinda hard to read when it's on one line, maybe do something like this above:

```js
const originSet = new Set();
for (const url of entityUrls) {
  const origin = UrlUtils.getOrigin(url);
  if (origin) originSet.add(origin);
}
```